services/vision/mlvision/classifier.go

```diff
@@ -57,6 +57,9 @@ func attemptToBuildClassifier(mlm mlmodel.Service) (classification.Classifier, e
 		}
 
 		confs := checkClassificationScores(probs)
+		if labels != nil && len(labels) != len(confs) {
```

Comment:
[supersupernit]: Since it can be probabilities or output0, might make sense to just say "length of output..."